tbdex/rfq/rfq_test.go

```diff
@@ -425,3 +430,25 @@ func TestVerify_SignedWithWrongDID(t *testing.T) {
 	assert.Error(t, err)
 	assert.Contains(t, err.Error(), "does not match message metadata from")
 }
+
+func TestIsValidNext(t *testing.T) {
+	pfiDID, _ := didjwk.Create()
+	walletDID, _ := didjwk.Create()
+	offeringID, _ := typeid.WithPrefix(offering.Kind)
+
+	r, _ := rfq.Create(
+		walletDID,
+		pfiDID.URI,
+		offeringID.String(),
+		rfq.Payin("100", "STORED_BALANCE"),
+		rfq.Payout("BANK_ACCOUNT"),
+	)
+
+	assert.False(t, r.IsValidNext(rfq.Kind))
+	assert.False(t, r.IsValidNext(order.Kind))
+	assert.False(t, r.IsValidNext(orderstatus.Kind))
+
+	// todo rfq can only be followed by quote... or close? 
```

Comment:
can rfq be followed by a quote or a close? or only a quote?

Reply:
rule of thumb is, close is valid after any message kind

Reply:
except for close itself! 🤓